v22.2/create-function.md

```diff
@@ -0,0 +1,110 @@
+---
+title: CREATE FUNCTION
+summary: The CREATE FUNCTION statement creates a user-defined function.
+toc: true
+keywords:
+docs_area: reference.sql
+---
+
+The `CREATE FUNCTION` [statement](sql-statements.html) creates a [user-defined function](user-defined-functions.html).
```

Comment:
sorry, one thing I forgot is that we probably want to mention that only `SELECT` statements are supported in UDF at the moment.

Reply:
That's covered here: https://deploy-preview-15075--cockroachdb-docs.netlify.app/docs/v22.2/user-defined-functions.html#overview

Reply:
yeah, that sounds good as well.